ui/app/views/fusor_ui/layouts/placeholder.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= content_for(:content) do %>
+  <%= render :partial => 'layouts/application_content' %>
+<% end %>
+
+<%= render :template => 'fusor_ui/layouts/placeholder_base' %>
```

Comment:
only difference from [application.html.erb](https://github.com/theforeman/foreman/blob/develop/app/views/layouts/application.html.erb#L5) is the different partial to refer to placeholder_base instead of layouts/base